backend/FwLite/FwLiteShared/Projects/LexboxProjectService.cs

```diff
@@ -64,24 +64,49 @@ public LexboxServer[] Servers()
         return Servers().FirstOrDefault(s => s.Id == projectData.ServerId);
     }
 
-    public async Task<FieldWorksLiteProject[]> GetLexboxProjects(LexboxServer server)
+    private static string CacheKeyForIsCrdtProject(LexboxServer server)
+    {
+        return $"IsCrdtProject|{server.Authority.Authority}";
+    }
+
+    public async Task<bool> IsCrdtProject(LexboxServer server, string projectCode)
+    {
+        return await cache.GetOrCreateAsync(CacheKeyForIsCrdtProject(server),
```

Comment:
I think let's not cache this since this is triggered by a user manually typing in the code, I can see a case where a project just became a CRDT project and now someone is trying to download it but it was previously cached as not a CRDT project because they checked too recently.

Reply:
Sure, that lets me get rid of the awkwardly-named CacheKeyForIsCrdtProject function too.